ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -223,7 +224,7 @@ function set_CA_for_nodes () {
 
 # Create the fixed ICSP for optional operators
 function create_settled_icsp () {
-    cat <<EOF | oc create -f -
+    cat <<EOF | oc apply --validate='ignore' -f -
```

Comment:
Why skip the schema validation here?

Reply:
This flag can disable the warning message below:
Warning: resource xxx/xxx is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by oc apply. oc apply should only be used on resources created declaratively by either oc create --save-config or oc apply. The missing annotation will be patched automatically.

Reply:
So, will it block the installation if use the `validate`? I don't think skip the validation is a good solution.